.github/nativefuncs.json

```diff
@@ -723,18 +717,6 @@
 			"helpText":"Returns whether or not a given path leads to a folder.",
 			"returnTypeString":"bool",
 			"argTypes":"string path"
-    },
-    {
-			"name":"NSPushUIPresence",
-			"helpText":"",
-			"returnTypeString":"void",
-			"argTypes":"struct presence"
-		},
-		{
-			"name":"NSGetMasterServerAuthResult",
-			"helpText":"",
-			"returnTypeString":"MasterServerAuthResult",
-			"argTypes":""
-		}
```

Comment:
accidental removal? causing compile check failure

Reply:
yeah, I wanted to fix it but didn't have time to.